docker/nugus_sim/run.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+from typing import Tuple
+
+import ruamel.yaml
+
+BINARIES_DIR = "/home/nubots/NUbots/binaries"
+CONFIG_DIR = BINARIES_DIR + "/config"
+
+ENV_VARS = {
+    "ROBOCUP_ROBOT_ID": "ROBOCUP_ROBOT_ID",
+    "ROBOCUP_TEAM_COLOR": "ROBOCUP_TEAM_COLOR",
+    "ROBOCUP_SIMULATOR_ADDR": "ROBOCUP_SIMULATOR_ADDR",
+}
+
+
+def read_args() -> Tuple[str, dict]:
+    if len(sys.argv) != 2:
+        print("Please specifiy a single role to run!")
+        sys.exit(1)
+
+    role = sys.argv[1]
+
+    built_roles = []
+
+    # Built roles
+    for filename in os.listdir(BINARIES_DIR):
+        if os.path.isfile(os.path.join(BINARIES_DIR, filename)):
+            built_roles.append(filename)
+
+    if role not in built_roles:
+        print("The role '" + role + "' does not exist!")
+        sys.exit(1)
+
+    # Read env vars
+    config = {k: os.environ.get(v) for k, v in ENV_VARS.items() if v in os.environ}
+
+    # Dict of env vars that didn't get set :(
+    unset_vars = {k: ENV_VARS[k] for k in ENV_VARS if k not in config}
+
+    # If not all needed environment variables were set
+    if len(unset_vars) != 0:
+        print("The following environment variables were not set!")
+        for var in unset_vars:
+            print(var)
+        sys.exit(1)
+
+    return role, config
+
+
+def set_env_vars(config: dict) -> None:
+    yaml = ruamel.yaml.YAML()
+    os.chdir(CONFIG_DIR)
+
+    # ROBOCUP_ROBOT_ID
+    file_name = "GlobalConfig.yaml"
+    with open(file_name) as file:
+        data = yaml.load(file)
+    data["player_id"] = int(config["ROBOCUP_ROBOT_ID"])
+
+    with open(file_name, "w") as file:
+        yaml.dump(data, file)
+
+    file_name = "GameController.yaml"
+    with open(file_name) as file:
+        data = yaml.load(file)
+    data["player_id"] = int(config["ROBOCUP_ROBOT_ID"])
+
+    with open(file_name, "w") as file:
+        yaml.dump(data, file)
+
+    # ROBOCUP_TEAM_COLOR
+    # ??
```

Comment:
If we don't use the team colour, does that mean we have an implicit mapping from team number to colour somewhere? @Bidski 
If so, we should make sure that that mapping is correct for this robocup.

Reply:
This should be set and used somewhere otherwise we are loading the wrong model